Userland/Libraries/LibWeb/HTML/Canvas/CanvasTransform.h

```diff
@@ -66,6 +76,24 @@ class CanvasTransform {
         transform(a, b, c, d, e, f);
     }
 
+    // https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-settransform-matrix
+    void set_transform(Geometry::DOMMatrix2DInit& init)
+    {
+        // 1. Let matrix be the result of creating a DOMMatrix from the 2D dictionary transform.
+        auto& realm = static_cast<IncludingClass&>(*this).realm();
+        auto maybe_matrix = Geometry::DOMMatrix::create_from_dom_matrix_2d_init(realm, init);
+        if (maybe_matrix.is_exception())
+            return;
```

Comment:
I don't see anything in the spec about silently swallowing exceptions here. Are we not supposed to throw them?

Reply:
Good point